innovation_sweet_spots/analysis/prototyping/public_discourse_analysis/Example_data_preprocessing.py

```diff
@@ -0,0 +1,130 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.3
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Clean text, build sentence corpus and analyse mentions over time
+#
+# - Basic cleaning of articles
+# - Extracting sentences (default context) and noun chunks for subsequent analysis
+# - Analyse mentions of search terms over time
+# - Appendix: alternative options for defining context around search terms
+
+# %% [markdown]
+# ## 1. Import dependencies
+
+# %%
+import os
+import pandas as pd
+import csv
+import spacy
+import pickle
+
+# %%
+# Change first element to location of project folder.
+os.chdir(os.path.join('/Users/jdjumalieva/Documents/Analysis/', 'innovation_sweet_spots'))
+
+# %%
+from innovation_sweet_spots.analysis.prototyping.public_discourse_analysis import pd_data_processing_utils as dpu
+from innovation_sweet_spots.getters.path_utils import OUTPUT_DATA_PATH
+
+# %%
+OUTPUTS_CACHE = OUTPUT_DATA_PATH / ".cache"
+DISC_OUTPUTS_DIR = OUTPUT_DATA_PATH / "discourse_analysis_outputs"
+DISC_OUTPUTS_DIR.mkdir(parents=True, exist_ok=True)
+
+# %%
+search_terms = ['heat pump', 'heat pumps']
+
+# %%
+nlp = spacy.load("en_core_web_sm")
+
+# %% [markdown]
+# ## 2. Clean text, build sentence corpus
+
+# %%
+# Read in article_text dataframe
+article_text = pd.read_csv(os.path.join(DISC_OUTPUTS_DIR, 'article_text_hp.csv'))
```

Comment:
You can also write it like this (a bit shorter, but perhaps a question of preference)
```suggestion
article_text = pd.read_csv(DISC_OUTPUTS_DIR / 'article_text_hp.csv')
```